### ISSUE TITLE
Downgrade Project files in test and lib folders

### ISSUE THREAD

**Issue by ArnoStrouwen** [cjdoris/julia-downgrade-compat-action#3]
Some Julia packages organize their tests using a Project file in the _test_ folder, e.g.:
https://github.com/SciML/Optimization.jl/blob/master/test/Project.toml
If I understand this action correctly, the additional Project file will not be downgraded.

Similarly, there are sometimes additional Project files in the _lib_ folder, e.g.:
https://github.com/SciML/Optimization.jl/blob/master/lib/OptimizationBBO/Project.toml

I think it would be a useful option to jointly downgraded all these Project files before testing.

**Comment by cjdoris** [comment]
Agreed. I think a good solution would be to add an argument to the action to specify the list of directories to look in. By default it would be `.` but in your example it would be `.,test,lib`. What do you think?

**Comment by cjdoris** [comment]
I'll implement it soon. BTW this action has moved to https://github.com/julia-actions/julia-downgrade-compat now. I'm about to archive this repo.
